ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -579,6 +583,16 @@ FindInfo findEntriesSince(FindToken token, long maxTotalSizeOfEntries) throws St
       boolean tokenWasReset = false;
       Offset logEndOffsetBeforeFind = log.getEndOffset();
       StoreFindToken storeToken = (StoreFindToken) token;
+      UUID remoteIncarnationId = storeToken.getIncarnationId();
+      if (!storeToken.getType().equals(StoreFindToken.Type.Uninitialized) &&
+          remoteIncarnationId != null && incarnationId != null && remoteIncarnationId.compareTo(incarnationId) != 0) {
+        // incarnationId mismatch, hence resetting the token to beginning
+        logger.info("Index : " + dataDir + " resetting offset after incarnation, new incarnation Id {}, "
```

Comment:
you seem to be using formatting for two out of three variables. Could you use formatting for all three?